### ISSUE TITLE
Baseline code

### ISSUE THREAD

**Issue by ddddyla** [twchen/lessr#6]
Hi. Could you share the baseline code that on your datasets, for example, SR-GNN? Because I cannot run the baseline code on your datasets.😭

**Comment by twchen** [comment]
For the code of baselines, please refer to the official implementations.
To run the baseline code on our datasets, you can convert our datasets to their format.
Besically, you just need to augment the sessions and store them as pickle files.
You may also need to shift the item IDs because our item IDs start from 0 and theirs may start from 1.

**Comment by ddddyla** [comment]
> For the code of baselines, please refer to the official implementations. To run the baseline code on our datasets, you can convert our datasets to their format. Besically, you just need to augment the sessions and store them as pickle files. You may also need to shift the item IDs because our item IDs start from 0 and theirs may start from 1.

Yes, I try to convert your datasets to their format, such as augment the sessions, shift item IDs and store them as pickle files. But the SR-GNN results I ran on your transformed dataset are quite different from those shown in your paper. So could you share your transformed datasets?
